dll/win32/shimgvw/lang/pt-PT.rc

```diff
@@ -0,0 +1,33 @@
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+/* Portuguese Language resource file
```

Comment:
Usually the comment header goes at the top -- just for your information.

Reply:
You're absolutely right, it was unintentional. Thanks